### PR TITLE
remove numpy dependency before setup can run

### PR DESCRIPTION
**Pull request by ericbuehl**
This clever hack is based on:
http://stackoverflow.com/questions/19919905/how-to-bootstrap-numpy-installation-in-setup-py

Without it, setup fails unless numpy is explicitly installed before setup is run.

### CODE REVIEW

File: setup.py [hunk 1]
```diff
@@ -122,5 +131,7 @@ def has_webp():
       ext_modules = ext_modules,
       package_dir = package_dir,
       package_data = package_data,
+      cmdclass = {'build_ext': build_ext},
+      setup_requires = ['numpy'],
```
Comment (xutaolvsc) [inline setup.py]: also add install_requires? since the code itself also depends on numpy

Comment (ericbuehl) [inline setup.py]: I'm considering that in another PR if this one is accepted

Comment (xutaolvsc) [inline setup.py]: I see. LGTM

**Comment by luispedro** [comment]
Thank you, this is good. Merging.

**Pull request merged**